setup.sh

```diff
@@ -505,6 +517,7 @@ do_setup() (
 
 	setup_osie
 	generate_certificates
+	configure_rsyslog_logging
```

Comment:
@Cbkhare @gianarb I thought we said we weren't going to make centralized logging a requirement? The docker-compose.yml was a sticky point but it doesn't seem too hard to handle that. Either with a `docker-compose-centralized-logging.yml` (template file) or with `sed` magic.

Reply:
I agree with you. My expectation as an outcome for the centralizing logging work was to get only solid documentation. Not sure how to review this PR.

Reply:
I would request some conclusion here as this change has taken some good amount of time and effort. 

Should I add the change to support logging without centralization (aggregation)?

Reply:
Though now we have the support of logging without centralization, This change of rsyslog would be required to provide the base configuration to the user if at some time he/she wishes to use the centralized logging. 

Conclusively, Tinkerbell has now a addon support for centralised logging along with the existing framework. And this centralized logging would also support plug and play model.